vendor-extra/LiberoPatternsBundle/src/Resources/views/text.html.twig

```diff
@@ -0,0 +1,17 @@
+{%- if nodes is iterable -%}
```

Comment:
`text` seems a generic name, as it can be confused with the article text for example; perhaps `nodes` or something related could be used as the template name (`all`? `join`?)

Reply:
Agree that the naming isn't great here (https://github.com/libero/pattern-library/pull/22#discussion_r240106726); one for the pattern library though.